integration_tests/_helpers.py

```diff
@@ -831,6 +837,25 @@ def _log_out(
     resp.raise_for_status()
 
 
+def _initiate_password_reset(
```

Comment:
```suggestion
def _forgot_password(
```

Reply:
this is backend, so `initiate` is actually more descriptive